<!-- <?xml version="1.0"?> -->
<launch>

    <node pkg="srosbag_pub" type="srosbag_pub" name="srosbag_pub" >
        <param name="bag_path" value="/home/zxj/data/rack_filter_opt/"/>
<!--
        <param name="bag_name"     value="Map201903151430"/>//该包的雷达点被车体盖板遮挡
-->
	<param name="bag_name" value="rack_de_300UL_0"/>
        <param name="time_scale" value="0.3"/>
<!--
        <param name="begin_play_time" type = "double" value="1573286200.563299"/>
-->
<!--
        <param name="begin_play_time" type = "double" value="1573360479.30518293"/>
-->
        <param name="begin_play_time" type = "double" value="0"/>
    </node>


</launch>
